client/src/main/java/com/kritsit/casetracker/client/domain/ui/controller/AdministratorController.java

```diff
@@ -58,6 +59,37 @@ public void initFrame(){
     
     public void initialize(){
         
+        deleteButton.setOnAction(event->{
+          TableViewSelectionModel<Staff> selection =  staffTable.getSelectionModel();
```

Comment:
With more actions being added these need to be broken up into their own methods even if just by calling a private method in this lambda